### PR TITLE
Fixed symlinks to directories not created as such on Windows.

### PR DESCRIPTION
**Pull request by Klaim**
See https://build2.org/article/symlinks.xhtml for details.

### CODE REVIEW

File: .gitattributes
Comment (Rookfighter) [inline .gitattributes]: Shouldn't this be `include/freetype/internal`?
Note: I tried your branch for SFML on Windows, bit it did not work, because of the unresolved `internal` symlink.

Comment (Klaim) [reply .gitattributes]: Yes I think I made a mistake when I made the commit - I got the list from a tool I wrote that list the symlinks to directories, I probably mis-copied the output.
I'll fix it. Did you try by adding `internal` see if it worked on your side?

Comment (Klaim) [reply .gitattributes]: My branch for sfml will not work I think even with this PR, I switched to this PR and dind't update the sfml one.

**Comment by Klaim** [comment]
BTW there is a bug in build2 stage version which prevents me from checking that version (auto-conf build-module not being found), it will have to wait next stage version.

Meanwhile, there is also an issue added for build2 about the symlinks annoyances (to help fix them): https://github.com/build2/build2/issues/189

File: .gitattributes
Comment (Rookfighter) [reply .gitattributes]: freetype works with these changes for me. Also have a look at the new SFML master, Windows build (including freetype) worked fully on my setup. Waiting for the CI build there.

Comment (Klaim) [reply .gitattributes]: I just tried locally and I see sfml master takes this branch and indeed it seems to work better - except like for this branch I hit the auto-conf build-module issue missing. I already reported this issue to the team so I think it will be fixed in stage soon.

**Comment by Klaim** [comment]
We just realized the issue I had with autoconf was due to something wrong with the install script when upgrading, so re-installing from scratch fixed the issue.

Both this branch and sfml package are fixed for me. @Rookfighter I'll let you merge once you confirm too.

**Comment by boris-kolpackov** [comment]
BTW, while we will try to merge this ASAP, you can sidestep the issue (and also get much faster experience) by using `libfreetype` as archive from here: https://queue.stage.build2.org/

FYI, this is a "staging queue" of packages that cannot be published to cppget.org because they require features only found in the staged version of the toolchain.

**Review by Rookfighter — APPROVED** [review]

**Comment by boris-kolpackov** [comment]
Merged with a few tweacks and a squash, thanks!

**Pull request closed without merging**
